README.md

```diff
@@ -1 +1,9 @@
-Assistop Repo
+# Assistop
+
+Created By:
+ - Dallas Griffith
+ - Paul Henson
+ - Jacob Pawek
+ - Garrett Welton
+
+Assistop is a IoT governer. It is designed to enhance the security of home assistance integrated within Smart Homes. When a user is not at home, it disables the assistants from being able to make critical changes.
```

Comment:
Grammer nazi nitpicks: 
Assistop is an* IoT governor*.
home assistants*

Reply:
Check newest commit -- i assume it will do this automatically